test/view/unique.cpp

```diff
@@ -95,7 +95,47 @@ int main()
         CONCEPT_ASSERT(ForwardRange<decltype(rng0)>());
         CONCEPT_ASSERT(!ForwardRange<decltype(rng0) const>());
         // ...and composable with unique_view
-        auto rng = rng0 | view::unique;
+        auto rng = rng0 | view::unique(equal_to{});
+        models<concepts::BidirectionalView>(aux::copy(rng));
+        models_not<concepts::RandomAccessRange>(rng);
+        models<concepts::BoundedRange>(rng);
+        models_not<concepts::SizedRange>(rng);
+        ::check_equal(rng, {4, 3, 2, 1});
+    }
+
+    {
+        auto const caseInsensitiveCompare = [](const std::string& s1, const std::string& s2){
+            if (s1.size() != s2.size())
+                return false;
+            for (unsigned i = 0; i < s1.size(); i++)
+                if (toupper(s1.at(i)) != toupper(s2.at(i)))
```

Comment:
Ahhh, `at`! It burns my eyes!

Reply:
Upss!!!
I didn't know that was not allowed here.
I guess it is because it can throw??

Reply:
No, it's mostly because I despise `at` due to the horrific effects it has on optimizers.